images/macos/tests/Android.Tests.ps1

```diff
@@ -8,10 +8,8 @@ Describe "Android" {
     $androidSdkManagerPackages = Get-AndroidPackages
     [int]$platformMinVersion = Get-ToolsetValue "android.platform_min_version"
     [version]$buildToolsMinVersion = Get-ToolsetValue "android.build_tools_min_version"
-    [string]$ndkLatestVersion = Get-ToolsetValue "android.ndk.latest"
-    [string]$ndkLtsVersion = Get-ToolsetValue "android.ndk.lts"
-    $ndkLatestFullVersion = (Get-ChildItem "$env:ANDROID_HOME/ndk/$ndkLatestVersion.*" | Select-Object -Last 1).Name
-    $ndkLtsFullVersion = (Get-ChildItem "$env:ANDROID_HOME/ndk/$ndkLtsVersion.*" | Select-Object -Last 1).Name
+    [array]$ndkVersions = Get-ToolsetValue "android.ndk.versions"
+    $ndkFullVersions = $ndkVersions | ForEach-Object { (Get-ChildItem "$env:ANDROID_HOME/ndk/${_}.*" | Select-Object -Last 1).Name }

```

Comment:
nitpick: 

`$ndkVersions | ForEach-Object { Get-ChildItem "$env:ANDROID_HOME/ndk/${_}.*" -Name | Select-Object -Last 1}`

Reply:
Fixed it